vars/helm.groovy

```diff
@@ -53,6 +53,8 @@ String generateModuleValues(String module_name, String module_version, Project p
         String repository
         if (custom_module || module_name == 'ui-bundle') {
             repository = Constants.ECR_FOLIO_REPOSITORY
+        } else if (module_name == 'mod-graphql' && module_version.contains("\\d{1,3}\\.\\d{1,3}\\.\\d{3,10}/")) {
+            repository = "folioci"
         } else {
             repository = module_version.contains('SNAPSHOT') ? "folioci" : "folioorg"
         }
```

Comment:
Also maybe better to refactor this code to switch case for example?
switch (module_name) {
    case 'ui-bundle':
    case custom_module:
        repository = Constants.ECR_FOLIO_REPOSITORY
        break
    case 'mod-graphql' && module_version ==~ /\d{1,3}\.\d{1,3}\.\d{3,10}/:
        repository = "folioci"
        break
    default:
        repository = module_version.contains('SNAPSHOT') ? "folioci" : "folioorg"
}